lib/src/executable.dart

```diff
@@ -200,7 +204,8 @@ transformers:
 ///
 /// If [error] is passed, it's used in place of the usage message and the whole
 /// thing is printed to stderr instead of stdout.
-void _printUsage([String error]) {
+void _printUsage(Configuration configuration, [String error]) {
+  var wordWrap = configuration?.wordWrap ?? (message) => message;
```

Comment:
I don't think you need the null-checks here anymore.

Reply:
done